pkg/theme/text.go

```diff
@@ -0,0 +1,395 @@
+package theme
+
+import (
+	"github.com/gookit/color"
+)
+
+var (
+	// DefaultTextColor is the default text color
+	DefaultTextColor = NewTextStyle(color.FgWhite, 0)
+	// DefaultHiTextColor is the default highlighted text color
+	DefaultHiTextColor = NewTextStyle(color.FgLightWhite, 0)
+
+	// SelectedLineBgColor is the background color for the selected line
+	SelectedLineBgColor = NewTextStyle(0, 0)
+
+	// SelectedRangeBgColor is the background color of the selected range of lines
+	SelectedRangeBgColor = NewTextStyle(0, 0)
+
+	OptionsFgColor = NewTextStyle(0, 0)
+
+	DiffTerminalColor = NewTextStyle(color.FgMagenta, 0)
+
+	FgCyan    = NewTextStyle(color.FgCyan, 0)
+	FgRed     = NewTextStyle(color.FgRed, 0)
+	FgGreen   = NewTextStyle(color.FgGreen, 0)
+	FgBlue    = NewTextStyle(color.FgBlue, 0)
+	FgYellow  = NewTextStyle(color.FgYellow, 0)
+	FgMagenta = NewTextStyle(color.FgMagenta, 0)
+)
+
+type TextColorType uint8
```

Comment:
I can't see where this is being used

Reply:
Jep this is a left over from a earlier iteration